src/parser/jobs/ast/DoTs.js

```diff
@@ -83,13 +85,13 @@ export default class DoTs extends Module {
 		// Suggestion for DoT clipping
 		const maxClip = Math.max(...Object.values(this._clip))
 
-		if (maxClip > 500) {
+		if (maxClip > MAXCLIP_THRESHOLD) {
 			this.suggestions.add(new Suggestion({
 				icon: ACTIONS.COMBUST_II.icon,
 				content: <Fragment>
 					Avoid refreshing <ActionLink {...ACTIONS.COMBUST_II} /> significantly before it expires.
 				</Fragment>,
-				severity: maxClip < 10000? SEVERITY.MINOR : maxClip < STATUS_DURATION[ACTIONS.COMBUST_II.id] ? SEVERITY.MEDIUM : SEVERITY.MAJOR,
+				severity: maxClip < MINOR_CLIPPING_THRESHOLD ? SEVERITY.MINOR : maxClip < STATUS_DURATION[ACTIONS.COMBUST_II.id] ? SEVERITY.MEDIUM : SEVERITY.MAJOR,
```

Comment:
Consider switching this to tiers when you get a chance.

Reply:
Roger that